handlers/log_chat.ts

```diff
@@ -30,6 +30,12 @@ composer.command("setlogchat", async (ctx) => {
     return;
   }
   try {
+    try {
+      const member = await ctx.api.getChatMember(ctx.chat.id, ctx.me.id);
```

Comment:
Just `ctx.getChatMember(ctx.me.id)`?

Reply:
That should not be included. It was just a test.